<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CancelDialog</class>
 <widget class="QDialog" name="CancelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отмена продажи</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/images/favicon.ico</normaloff>:/images/favicon.ico</iconset>
  </property>
  <widget class="QDialogButtonBox" name="DialogBtnBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="SoftLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="SoftLabel">
      <property name="text">
       <string>ПО:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="SoftComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>451</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="CustomerLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="CustomerLabel">
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>37</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Клиент:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="CustomerComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>188</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="TypeLayout">
    <item>
     <widget class="QLabel" name="TypeLabel">
      <property name="text">
       <string>Тип:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TypeEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>451</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="SerialLayout">
    <item>
     <widget class="QLabel" name="SerialLabel">
      <property name="text">
       <string>Серийный номер:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SerialEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ActDateLayout">
    <item>
     <widget class="QLabel" name="ActDateLabel">
      <property name="text">
       <string>Дата активации:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ActDateEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>CancelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>CancelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
